media/css/mozorg/technology.scss

```diff
@@ -878,11 +807,11 @@ html[dir="rtl"] #technologies > section {
         }
 
         p {
-            @include font-size(16px);
+            @include font-size-level5;
```

Comment:
Yeah, I think if we change anything now then a fixed 16px rule sounds like the best bet. Totally up to you though 👍